### DISCUSSION TITLE
Phantom count of repositories

### DISCUSSION THREAD

**Discussion by DeiAsPie** [community/community#33685] [Other Feature Feedback, Questions, & Ideas] [answered]
![Screenshot](https://user-images.githubusercontent.com/93835541/191480040-359bc629-5578-4cdd-912f-286448e1fe3a.png)

Although I have no starred repos yet it somehow happens to count 23 of them. Could someone point to me a possible solution or the cause of this ?

**Comment by Matteo-it** [comment, marked as answer]
It's for sure a Bug, contact the support center and they will fix it: https://support.github.com/

**Comment by EnochianD** [comment]
![GIF_20220927_053854.gif](https://user-images.githubusercontent.com/110375788/192498676-d6d0bf52-963e-4e66-a256-8014905614f7.gif)

**Reply by KristieSandra222** [reply]
Hi

**Reply by Ryanpatrickmac** [reply]
- [ love```

```

**Comment by monjie110** [comment]
> It's for sure a Bug, contact the support center and they will fix it: https://support.github.com/
